src/modules/emote_menu/components/Sidebar.jsx

```diff
@@ -48,7 +48,13 @@ export default function Sidebar({section, onClick, categories}) {
 
   const renderDraggable = useDraggableInPortal();
   useEffect(() => setMiddleCategories(categories.middle), [categories.middle]);
-  useAutoScroll(section, containerRef, [...categories.top, ...middleCategories, ...categories.bottom]);
+
+  useAutoScroll(
+    section,
+    containerRef,
+    [...categories.top, ...middleCategories, ...categories.bottom],
+    emojiButtonHidden ? WindowHeight + 49 : WindowHeight // 1px (divider) + 48px (emojiButton) = 49px
```

Comment:
shouldn't WindowHeight be `WINDOW_HEIGHT`? only enums are PascalCase

Reply:
also you can optimize this a bit: `WINDOW_HEIGHT + (emojiButtonHidden ? 49 : 0)`